src/test/java/org/candlepin/resource/EntitlementResourceTest.java

```diff
@@ -392,12 +392,12 @@ public void testUpdateEntitlementWith0Quantity()
             entResource.updateEntitlement(entitlementId, entitlementDTO)
         );
 
-        verify(entitler, never()).adjustEntitlementQuantity(any(Consumer.class), any(Entitlement.class), any(Integer.class));
+        verify(entitler, never())
+                .adjustEntitlementQuantity(any(Consumer.class), any(Entitlement.class), any(Integer.class));
```

Comment:
Use 4-space indentation on these continuations

Reply:
Updated to 4 spaces